src/ngraph/graph_util.hpp

```diff
@@ -132,5 +134,8 @@ namespace ngraph
 
     bool is_one(std::shared_ptr<Node> reduce_constant);
 
-    bool is_used(std::shared_ptr<Node> node);
+    bool is_used(Node* node);
```

Comment:
This could use a short comment too, since there are different ways for a node to be used.